src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx

```diff
@@ -0,0 +1,128 @@
+import { Button } from "primereact/button";
+import { InputText } from "primereact/inputtext";
+import { Card } from "primereact/card";
+import { Dropdown } from "primereact/dropdown";
+import { useState } from "react";
+import { AnswerItem } from "./components/answer-item";
+
+import { observer } from "mobx-react";
+import { QuizStore } from "../store/CreateQuizStore";
+
+export const CreateQuizSecondCard: React.FC = observer(() => {
+  const [value, setValue] = useState<string>("");
+
+  const [selectedCity1, setSelectedCity1] = useState(null);
+  const cities = [
+    { name: "New York", code: "NY" },
+    { name: "Rome", code: "RM" },
+    { name: "London", code: "LDN" },
+    { name: "Istanbul", code: "IST" },
+    { name: "Paris", code: "PRS" },
+  ];
+
+  const onCityChange = (e: { value: any }) => {
+    setSelectedCity1(e.value);
+  };
+
+  const [list, setList] = useState([1, 2]);
```

Comment:
If you store the list of answers as indexes you run the risk to have duplicates when you do a series of delete-and-create new answers as the list could have the same length multiple times. I suggest to store the list as an array of either objects or strings with the actual values to be able to identify them better. If this is just Work In Progress then is fine, just don't forget to change it.